sheet_to_triples/read_csv.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 Visual Meaning Ltd
+# This is free software licensed as GPL-3.0-or-later - see COPYING for terms.
+
+"""Loading tabular data from csv files."""
+
+import csv
+
+
+class Cell:
+    """Class wrapper for row value."""
+
+    def __init__(self, value):
+        self.value = value
+
+    def value(self):
+        return self.value
+
+class Book:
+    """Wrapper around csv reader object to present common interface."""
+
+    def __init__(self, book):
+        self._book = book
+
+    def __repr__(self):
+        return f'{self.__class__.__name__}({self._book})'
+
+    @classmethod
+    def from_path(cls, path):
+        csv_file = open(path, 'r')
+        return cls(csv.reader(csv_file))
```

Comment:
Probably better to just hold onto the csv path in the class, and open/read through the file in `iter_rows_in_sheet` - does mean a missing file error happens a bit later, but easier to handle file closing.